airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -2,6 +2,7 @@ connector_image: airbyte/source-amazon-seller-partner:dev
 tests:
   spec:
     - spec_path: "integration_tests/spec.json"
+    - backward_compatibility_tests_config.disable_for_version: "0.2.26"
```

Comment:
This isn't the correct place to put. @natalyjazzviolin search other examples of how add the `backward_compatibility_tests_config` probably will be there one for discovery

Reply:
Yup! Realized my mistake!